app/controllers/webhook_controller.rb

```diff
@@ -21,18 +20,13 @@ def callback
     events = client.parse_events_from(body)
     events.each { |event|
       case event
-      when Line::Bot::Event::Message
-        case event.type
-        when Line::Bot::Event::MessageType::Text
-          message = {
-            type: 'text',
-            text: event.message['text']
-          }
-          client.reply_message(event['replyToken'], message)
-        when Line::Bot::Event::MessageType::Image, Line::Bot::Event::MessageType::Video
-          response = client.get_message_content(event.message['id'])
-          tf = Tempfile.open("content")
-          tf.write(response.body)
+      when Line::Bot::Event::Follow
+        user_id = event['source']['userId']
+        if !(Customer.exists?(user_id: user_id)) && user_id.present?
```

Comment:
[nits]
かっこなしでも動きますっ
```suggestion
        if !Customer.exists?(user_id: user_id) && user_id.present?
```